src/compiler/checker.ts

```diff
@@ -1237,10 +1239,10 @@ module ts {
                     }
                 }
 
-                function writeTypeofSymbol(type: ObjectType) {
+                function writeTypeofSymbol(type: ObjectType, flags?: TypeFormatFlags) {
                     writeKeyword(writer, SyntaxKind.TypeOfKeyword);
                     writeSpace(writer);
-                    buildSymbolDisplay(type.symbol, writer, enclosingDeclaration, SymbolFlags.Value);
+                    buildSymbolDisplay(type.symbol, writer, enclosingDeclaration, SymbolFlags.Value, undefined, flags);

```

Comment:
now that I'm passing a TypeFormatFlags parameter in here as well, does it make sense to change the parameter name from "flags" to "symbolFormatFlags" to make it less confusing?